src/modules/Core/selectors.js

```diff
@@ -3,6 +3,8 @@
 import type { EdgeCurrencyWallet } from 'edge-core-js'
 
 import type { State } from '../ReduxTypes'
+import { convertCurrency } from '../UI/selectors.js'
+import { getDefaultFiat, getDefaultIsoFiat } from '../UI/Settings/selectors.js'
 
```

Comment:
`convertCurrency` will remain the name of the method we use to call the core, but will be moved out of the `core` part of the file system